csrc/executor_utils.cpp

```diff
@@ -369,6 +369,13 @@ std::unique_ptr<caching::VectorizedTensorInfo> getVectorizedTensorValidationInfo
 void validateAlignedVectorizeExtents(
     const VectorizedSetInfo& info,
     ExpressionEvaluator& expr_eval) {
+  if (info.contig_alloc_ids.empty()) {
+    // This happens when device lowering removes the `Expr` that computes
+    // `info.consumer_tv` because it's merely an alias.
+    // `getTensorIndexFromIdGraph` captures only remaining `Expr`s.
+    return;
+  }
+
   NVF_ERROR(
```

Comment:
Is this redundant with the above `if (info.contig_alloc_ids.empty()) {`?

Reply:
Done!